### PR TITLE
Fixes to customerReview controller

### PR DESCRIPTION
**Pull request by akiutoslahti**
GET /all is not used.
Added GET / and DELETE /:id for e2e testing.

### CODE REVIEW

File: controllers/customerReview.js [hunk 1]
```diff
@@ -153,4 +153,34 @@ customerReviewRouter.get('/:id', async (req, res) => {
...
+  try {
+    const customerReview = await db.CustomerReview.findOne({
+      where: { id: req.params.id }
+    })
+
+    if (!customerReview) {
+      return res.status(204).send()
+    }
+
+    console.log(customerReview)
```
Comment (cxcorp) [inline controllers/customerReview.js]: probably want to delete this

**Comment by cxcorp** [comment]
Was `GET /all` used by admin manually?

**Comment by akiutoslahti** [comment]
> Was `GET /all` used by admin manually? IIRC @Ha3a is doing some work related to these endpoints right now

GET / can be used no instead. It has the same function but wraps everything neatly to object (needed for iteration).

File: controllers/customerReview.js (hunk 1 above)
Comment (akiutoslahti) [reply controllers/customerReview.js]: done

**Comment by cxcorp** [comment]
We decided that @Ha3a 's endpoint `/all` will just be renamed to `/by-group` or whatever and we keep this one for the tests.

**Pull request merged**